photon-client/src/components/dashboard/tabs/PnPTab.vue

```diff
@@ -26,11 +26,8 @@ const interactiveCols = computed(
         { name: '2020 High Goal Outer', value: TargetModel.InfiniteRechargeHighGoalOuter },
         { name: '2020 Power Cell (7in)', value: TargetModel.CircularPowerCell7in },
         { name: '2022 Cargo Ball (9.5in)', value: TargetModel.RapidReactCircularCargoBall },
-        { name: '200mm AprilTag', value: TargetModel.Apriltag_200mm },
-        { name: '6in (16h5) Aruco', value: TargetModel.Aruco6in_16h5 },
-        { name: '6in (16h5) AprilTag', value: TargetModel.Apriltag6in_16h5 },
-        { name: '6.5in (36h11) Aruco', value: TargetModel.Aruco6p5in_36h11 },
-        { name: '6.5in (36h11) AprilTag', value: TargetModel.Apriltag6p5in_36h11 }
+        { name: '2023 AprilTag 6in (16h5)', value: TargetModel.kAprilTag6in_16h5 },
```

Comment:
Do we still have the tag family drop-down in the UI that needs to be set? If so, looks like we need to remove it now?

Reply:
Wdym? Also, PNPTab is dead code.

Reply:
On the apriltag tab, is there a drop-down for tag family?

Reply:
Yes, does the tag family settings prop no longer exist?